sos/report/plugins/microk8s.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2023 Canonical Ltd.,
+#                    David Negreira <david.negreira@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Microk8s(Plugin, UbuntuPlugin):
+
+    short_desc = 'The lightweight Kubernetes'
+    plugin_name = "microk8s"
+    profiles = ('container',)
+
+    packages = ('microk8s',)
+
+    microk8s_cmd = "microk8s"
+
+    def setup(self):
+        self.add_journal(units="snap.microk8s.*")
+
+        microk8s_subcmd = [
+                'addons repo list', 'config',
+                'ctr plugins ls', 'ctr plugins ls -d',
+                'status', 'version']
+
+        for subcmd in microk8s_subcmd:
+            self.add_cmd_output('%s %s' % (self.microk8s_cmd, subcmd))
```

Comment:
f-strings are recommended for string interpolation, and this is a bit more readable with list comprehension:

```python
self.add_cmd_output([
    f"microk8s {subcmd}" for subcmd in microk8s_subcmds
])
```

Reply:
Fixed on https://github.com/sosreport/sos/commit/cab5ce44636a380aac1d2443923acd6e564bc3a0